sections/browsers.include

```diff
@@ -2910,26 +2910,34 @@
     a previous state the user does not go back in time, and therefore it would be inappropriate to
     put the time in the session history title.
 
-    <pre highlight="html">
-  &lt;!DOCTYPE HTML>
-  &lt;TITLE>Line&lt;/TITLE>
-  &lt;SCRIPT>
-  setInterval(function () { document.title = 'Line - ' + new Date(); }, 1000);
-  var i = 1;
-  function inc() {
-    set(i+1);
-    history.pushState(i, 'Line - ' + i);
-  }
-  function set(newI) {
-    i = newI;
-    document.forms.F.I.value = newI;
-  }
-  &lt;/SCRIPT>
-  &lt;BODY ONPOPSTATE="set(event.state)">
-  &lt;FORM NAME=F>
-  State: &lt;OUTPUT NAME=I>1&lt;/OUTPUT> &lt;INPUT VALUE="Increment" TYPE=BUTTON ONCLICK="inc()">
-  &lt;/FORM>
-  </pre>
+    <xmp highlight="html">
+      <!DOCTYPE html>
+      <title>Line</title>
+      <script>
+        setInterval(function() {
+          document.title = 'Line - ' + new Date();
+        }, 1000);
```

Comment:
(I would indent this line since it is part of the setInterval... but that might be an odd idiosyncracy of mine)